<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1186</width>
    <height>737</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label_pic2">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="3" column="0" alignment="Qt::AlignLeft">
     <widget class="QPushButton" name="Nearest_Neighbor_Interpolation">
      <property name="text">
       <string>Nearest Neighbor Interpolation</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0" alignment="Qt::AlignLeft">
     <widget class="QPushButton" name="Histogram_Equalization">
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Histogram Equalization</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QLabel" name="label_pic">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="maximumSize">
       <size>
        <width>512</width>
        <height>512</height>
       </size>
      </property>
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="5" column="0" alignment="Qt::AlignLeft">
     <widget class="QPushButton" name="Local_Histogram">
      <property name="text">
       <string>Local Histogram</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1186</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>Nearest_Neighbor_Interpolation</sender>
   <signal>clicked()</signal>
   <receiver>label_pic</receiver>
   <slot>setFocus()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>219</x>
     <y>595</y>
    </hint>
    <hint type="destinationlabel">
     <x>223</x>
     <y>546</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Histogram_Equalization</sender>
   <signal>clicked()</signal>
   <receiver>label_pic</receiver>
   <slot>raise()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>422</x>
     <y>670</y>
    </hint>
    <hint type="destinationlabel">
     <x>425</x>
     <y>509</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
